src/service/service.ts

```diff
@@ -283,6 +287,37 @@ export class Discv5 extends (EventEmitter as { new (): Discv5EventEmitter }) {
     });
   }
 
+  /**
+   * Broadcast TALKREQ message to all nodes in routing table
+   */
+  public async sendTalkReq(payload: Buffer, protocol: string): Promise<void> {
```

Comment:
maybe we can change the name of this method to something that more clearly evokes 'sending messages to all nodes'.
Eg: `broadcastTalkReq`.

I'd imagine that we would eventually also have a `sendTalkReq(srcId: NodeId, protocol: string, payload: Buffer): Promise<void>` that would mirror `sendTalkResp` for sending a TALKREQ to a single peer.

Reply:
Can we use `Uint8Array` instead of `Buffer` for these added function params? Here and in the other added methods/functions. We can use `Buffer` internally, but it's nice to allow consumers to pass in native `Uint8Array`, rather than need to wrap in a `Buffer`.